gamedata/scripts/magazine_binder.script

```diff
@@ -452,29 +468,34 @@ function validate_wep(id, sec)
 	local w_bt = get_weapon_base_type(sec)
 	local m_bt = get_magazine_base_type(mag_data.section)
 	if w_bt ~= m_bt then
-		print_dbg("Incompatible magazine %s (bt %s) found on gun %s (bt %s). Granting default magazine.", mag_data.section, m_bt, sec, w_bt)
+		print_err("(CLEANUP) Incompatible magazine %s (bt %s) found on gun %s (typ %s, bt %s). Granting default magazine.", mag_data.section, m_bt, id, sec, w_bt)
 		mag_data.section = weapon_default_magazine(obj)
 		empty_table(mag_data.loaded)
```

Comment:
say this gun had 10 rounds in it what happens if you fire this gun now? will all those rounds disappear?
same question for ejecting the magazine.